code/game/objects/items/weapons/twohanded.dm

```diff
@@ -819,6 +819,8 @@
 		var/mob/living/carbon/human/user = loc
 		if(user.species.exotic_blood)
 			user.vessel.remove_reagent(user.species.exotic_blood,2)
+		else if(user.get_species() == "Machine")
+			user.oxyloss += 0.368

```

Comment:
Directly setting damage levels (not using `adjustFooLoss`) leads to bad things like the health dummy not updating